contrib/local/ibc/gaiad.Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM golang:1.18-alpine3.16 AS go-builder
```

Comment:
can we use plain golang image, rather than alpine images?
this also reduces our build time

Reply:
This is just the builder image, tbh, we dont even use go from here, since we download the binary directly. This was taken just to keep all our gaia based dockerfiles consistent, similar to our persistenceCore dockerfile.

Reply:
Lets create and issue for this and try to add this as a feature request.
For this PR, as long as we are able to create a single binary that is statically linked and working, should be the scope of this PR